app/controllers/api/v1/events_controller.rb

```diff
@@ -0,0 +1,69 @@
+# :reek:InstanceVariableAssumption
+# :reek:NilCheck
+
+class Api::V1::EventsController < ApplicationController
+  before_action :authenticate_user!, only: %i[create update destroy]
+  before_action :verify_organizer, only: %i[create update destroy]
+  before_action :set_events, only: %i[index]
+  before_action :set_event, only: %i[show update destroy]
+  before_action :set_organization_event, only: %i[update destroy]
+
+  def index
+    render json: @events
+  end
+
+  def show
+    return head :forbidden unless @event.social? || (current_user&.organization&.events&.find_by(id: @event.id))
```

Comment:
the 2nd condition is really hard to read :)

Reply:
@anatoliliotych  and what can i do with this? move to new method?